app/models/post_tag.rb

```diff
@@ -0,0 +1,20 @@
+# == Schema Information
+#
+# Table name: post_tags
+#
+#  id         :bigint           not null, primary key
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#  post_id    :bigint
+#  tag_id     :bigint
+#
+# Indexes
+#
+#  index_post_tags_on_post_id             (post_id)
+#  index_post_tags_on_post_id_and_tag_id  (post_id,tag_id) UNIQUE
+#  index_post_tags_on_tag_id              (tag_id)
+#
+class PostTag < ApplicationRecord
+  belongs_to :post
+  belongs_to :tag
```

Comment:
There's might be a case when multiple PostTag records contain the same pair post_id and tag_id.
Please add rails validation for pair uniqueness